src/server/schema/resolvers/types/Life.ts

```diff
@@ -0,0 +1,14 @@
+import { GraphQLLifeResolvers } from '../definitions';
+
+const LifeGraphQL: GraphQLLifeResolvers = {
+    id: root => root._id,
+    firstName: root => root.firstName,
+    lastName: root => root.lastName,
+    fullName: root => root.firstName + " " + root.lastName,
+    title: root => root.title,
+    description: root => root.description,
+    birthday: root => root.birthday,
+    hobbies: root => root.hobbies
```

Comment:
you can remove for others and just do for `id` and `fullName`
Others are done by default already since the field names are the same